libraries/terminal/src/test/AnsiEscape.test.ts

```diff
@@ -1,26 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-// Explicitly use the colors package here instead of Colorize
-import colors from 'colors/safe';
 import { AnsiEscape } from '../AnsiEscape';
+import { Colorize } from '../Colorize';
 
 describe(AnsiEscape.name, () => {
-  let initialColorsEnabled: boolean;
-
-  beforeAll(() => {
-    initialColorsEnabled = colors.enabled;
-    colors.enable();
-  });
-
-  afterAll(() => {
-    if (!initialColorsEnabled) {
-      colors.disable();
-    }
-  });
-
   it('calls removeCodes() successfully', () => {
-    const coloredInput: string = colors.rainbow('Hello, world!');
+    const coloredInput: string = Colorize.green('Hello, world!');
```

Comment:
Sidenote: We should have a rainbow option as well. Is it overkill? Yes. Is it neat? Also yes.

Reply:
Done.